examples/standalone/cli/README.md

```diff
@@ -0,0 +1,30 @@
+# Standalone Analyzer CLI Example
```

Comment:
In the `examples/*` directory, I have 2 example implementations:
1. `cli` - build and call a simple CLI (the toy build from Abhi's example [here](https://github.com/codecov/codecov-javascript-bundler-plugins/pull/71)!)
2. `library-import` - build a simple script that calls our new `standalone-analyzer` library